cagov/ui-claim-tracker
# Remove dotenv package

## Ticket

Closes #292 

## Changes

- Removes [dotenv](https://github.com/motdotla/dotenv) as a dependency 

## Context

Next.js has built-in support for [environment variables](https://nextjs.org/docs/basic-features/environment-variables), so we don't need to use a separate package for that.

⚠️ This requires a change for local development: **Rename .env to .env.local** ⚠️ 

Technically, Next.js will still load `.env`, but since `.env.local` is safe for secrets, we should use that.

## Testing

1. Check out this branch
2. Rename your `.env` file to `.env.local` 
3. `yarn dev`
3. Confirm your env vars are still loading (no error: `Error: EISDIR: illegal operation on a directory, read`)
